Poker.BE/Poker.BE.Domain/Game/Deck.cs

```diff
@@ -26,6 +26,7 @@ public class Deck
         private Card[] cards;
         private int shuffleTimes;
         private Random random;
+        private int cardIndex;
```

Comment:
The deck is an array of cards.
every time we pull a card we need to increase the card index (so we won't pull the same card each time).